islandora_matomo.module

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Contains islandora_matomo.module.
+ */
+
+function islandora_matomo_preprocess_node(array &$variables) {
+  \Drupal::logger('islandora_matomo')->info(t("Node view mode @mode.", ['@mode' => $variables['view_mode']]));
+  if (!$variables["teaser"]) {
```

Comment:
do you think this is being applied too generally? like i'm concerned about the overhead where audio and video nodes are shown without the viewer (ie search results, latest additions, recent items)

also what happens when there are multiple nodes on a page? the js isn't added multiple times, right?

and should this only be applied on service files?